www/layouts/about/section.html

```diff
@@ -358,7 +358,7 @@ <h1 class="text-center-mobile pb-5">Beyond Course Materials</h1>
               Open Learning Library is always open for self-guided learning, and 
               does not include live support, discussion forums, or certificates of completion.
             </p>
-            <a class="red-btn" href="/pages/open-learning-library/">Explore Open Learning Library courses </a>
+            <a class="red-btn" href="/course-lists/open-learning-library/">Explore Open Learning Library courses </a>
```

Comment:
I think you can undo this change and rebase your branch with main as this link: `Explore Open Learning Library courses` was updated in this [PR](https://github.com/mitodl/ocw-hugo-themes/pull/588) yesterday

Reply:
I think you can undo this change as the correct link was provided by Daniel (as shown below) for which a PR (mentioned in above comment) was created and merged:
<img width="630" alt="image" src="https://user-images.githubusercontent.com/93309234/161523825-b638340c-fa4b-4b0d-adfe-1612b61d0221.png">

Reply:
Please use https://ocw.mit.edu/collections/mit-open-learning-library/